scenario/node/src/gatewaycontext.ts

```diff
@@ -84,38 +84,68 @@ export class GatewayContext {
         const event: ChaincodeEvent = await this.getChaincodeEventListener(listenerName).next();
         return event;
     }
-
```

Comment:
You lost a blank line between method definitions. It would be good to have it back for readability